src/common/readfield_fi.f90

```diff
@@ -196,12 +196,14 @@ subroutine readfield_fi(istep, nhleft, itimei, ihr1, ihr2, itimefi, ierror)
 
     timepos = iavail(ntav2)%timePos
     timeposm1 = timepos ! Default: No deaccumulation possible
-    if (ntav1 /= 0) then
-      if (iavail(ntav2)%fileNo .eq. iavail(ntav1)%fileNo) then
-        ! previous timestep in same file for deaccumulation
-        timeposm1 = iavail(ntav1)%timePos
-      endif
+    if ((ntav1 /= 0) .and. ((iavail(ntav2)%fileNo .eq. iavail(ntav1)%fileNo))) then
```

Comment:
Must split into two lines, order of evaluation is not determined (iavail(ntav1) can be undefined)

Reply:
removed that code completely, is no longer needed